pylib/anki/_legacy.py

```diff
@@ -98,6 +107,37 @@ def register_deprecated_attributes(
         }
 
 
+class DeprecatedNamesMixinStandalone(DeprecatedNamesMixin):
```

Comment:
"DeprecatedModuleNamesMixin"? "DeprecatedNamesMixinForModule"? Open to other ideas :-)